packages/terra-form/CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ Unreleased
 ### Removed
 * Removed nightwatch.config file
 
+### Breaking Changes
+
+* Select and SelectField no longer accepts a list of strings, but a list of objects to set their value/display
```

Comment:
`value/display` a little misunderstanding. It seems value or display, but it's the value and display value pair. Could we update it and make it clear?